.github/workflows/continous-integration.yml

```diff
@@ -540,15 +540,22 @@ jobs:
           coverage combine "${final_dir}/"*
           coverage xml
 
-      - name: Upload reports to codeclimate
-        if: needs.changes.outputs.backend == 'true'
-        uses: paambaati/codeclimate-action@b649ad206d2e83dafb9ed130deba698aa1b41d78
+      - name: Analyse code with SonarCloud
+        uses: actions/checkout@ac593985615ec2ede58e132d2e21d2b1cbd6127c
+        with:
+          # Disabling shallow clone is recommended for improving relevancy of reporting
+          fetch-depth: 0
```

Comment:
This looks like a duplicate of the first repo checkout step, should we add the `with` over there in L511

Reply:
You're right! Updated.